content/en/blog/extend-falco-inputs-with-a-plugin-register.md

```diff
@@ -0,0 +1,107 @@
+
+# Details of your plugin
+
+In this section, we'll describe the key elements for describing your plugin.
+
+## License
+
+The license is not mandatory and you're free to choose the one you want for your plugin. Keep in mind it's often a good practice to have a license.
+
+## ID
+
+Every source plugin requires its own, unique plugin event `ID` to interoperate with `Falco` and the other plugins. This `ID` is used in the following ways:
+
+* The `ID` is saved in in-memory event objects and is used to identify the associated plugin that injected the event.
+* The `ID` is saved in capture files and is used to recreate in-memory event objects when reading capture files.
+
+The `ID` must be unique to ensure that events written by a given plugin will be properly associated with that plugin (and its event sources, see below).
+
+## Name
+
+Each plugin in the registry must have its own `name` and can be different of `event source`, that can be shared accross different plugins (eg, for k8s audit logs, we may have different plugins but only one type of `event source`).
+
+The `name` should match this regular expression `^[a-z]+[a-z0-9_]*$`.
+
+## Fields
+
+The `fields` are used for conditions in rules, please describe the available fields of your plugin in the README.
+
+For example:
+
+| Name                     | Type   | Description                    |
+| ------------------------ | ------ | ------------------------------ |
+| `docker.status`          | string | Status of the event            |
+| `docker.id`              | string | ID of the event                |
+| `docker.from`            | string | From of the event (deprecated) |
+| `docker.type`            | string | Type of the event              |
+| `docker.action`          | string | Action of the event            |
+| `docker.stack.namespace` | string | Stack Namespace                |
+
+# Propose your Plugin
+
+It's easy, follow these steps:
+* fork https://github.com/falcosecurity/plugins
+* update `https://github.com/falcosecurity/plugins/blob/master/registry.yaml` in your fork
+* add your plugin in the `plugins.source` list
+* submit the Pull Request with your modification
+
+# Share with the community
+
+By sharing your idea and code with the community, you will have direct feedback. Don't hesitate to share your work in the Slack kubernetes#falco before submitting your PR too.
+
+# Conclusion
+
+With this post, you should know how to propose your plugin to the community and have feedbacks about. The following posts will describe more advanced use-cases with events from Cloud Services. Stay tuned :wink:.
```

Comment:
Suggested correction: and have **feedback** about **it**.
Suggested corrections: emojis like :wink:, :tada:, or :warning: aren't rendered when publishing the blog.